databricks/koalas/frame.py

```diff
@@ -258,6 +258,53 @@ def to_html(self, buf=None, columns=None, col_space=None, header=True, index=Tru
         return validate_arguments_and_invoke_function(
             kdf.to_pandas(), self.to_html, pd.DataFrame.to_html, args)
 
+    def to_string(self, buf=None, na_rep='NaN', float_format=None, header=True,
+                  index=True, length=False, dtype=False, name=False,
+                  max_rows=None):
+        """
+        Render a string representation of the Series.
+
+        .. note:: This method should only be used if the resulting Pandas DataFrame is expected
```

Comment:
Actually, we're mixing NumPy docstring `Paramters` with underscores, and typical sphinx docstring (like `.. note::`) ..

If I am not mistaken, we can do as:

```

        Notes
        -----
        This method should only be used if the resulting Pandas DataFrame is expected
```

Reply:
Ah ok. They both work. We have other places too. I'm thinking we should just submit a separate PR to address all of them at the same time. Sounds good?

Reply:
actually i copied the series one haha. going to fix it.